backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -41,6 +47,25 @@
     private BacktraceDatabaseSettings databaseSettings;
     private boolean _enable = false;
 
+    /**
+     * Add attributes to native reports
+     * @param name attribute name
+     * @param value attribute value
+     */
+    public native void AddAttribute(String name, String value);
```

Comment:
in Java it is normal to start the name of a method from a lower case letter

Reply:
renamed to lowercase - other methods as well.